README.md

```diff
@@ -1,73 +1,76 @@
 </p>
 
 ### How can I use this in my monorepo?
 
-You can use the Turbo Cache Server as **Github Action**. Here is how:
-
-1. On your workflow files, add the following global environment variables:
-
-```yml
-env:
-  TURBO_API: "http://127.0.0.1:8585"
-  TURBO_TEAM: "NAME_OF_YOUR_REPO_HERE"
-  # The value of TURBO_TOKEN is irrelevant
-  # as we don't perform any auth against the cache server
-  # but it's still necessary for Turborepo
-  TURBO_TOKEN: "turbo-token"
-```
-
-> [!NOTE] These environment variables are required by Turborepo so it can call
-> the Turbo Cache Server with the right HTTP body, headers and query strings.
-> These environment variables are necessary so the Turborepo binary can identify
-> the Remote Cache feature is enabled and can use them across all steps. You can
-> [read more about this here](https://turbo.build/repo/docs/ci#setup) on the
-> Turborepo official docs.
-
 Make sure that you have an S3-compatible storage available. We currently tested
 with:
 
 - [Amazon S3](https://aws.amazon.com/s3/)
 - [Cloudflare R2](https://www.cloudflare.com/en-gb/developer-platform/r2/)
 - [Minio Object Storage](https://min.io/)
 
-1. Still on your `yml` file, after checking out your code, use our custom action
-   to start the Turbo Cache Server in the background:
-
-```yml
-- name: Checkout repository
-  uses: actions/checkout@v4
-
-- name: Turborepo Cache Server
-  uses: brunojppb/turbo-cache-server@1.0.3
-  env:
-    PORT: "8585"
-    S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
-    S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
-    S3_ENDPOINT: ${{ secrets.S3_ENDPOINT }}
-    S3_BUCKET_NAME: your-bucket-name-here
-    # Region defaults to "eu-central-1"
-    S3_REGION: "eu-central-1"
-    # if your S3-compatible store does not support requests
-    # like https://bucket.hostname.domain/. Setting `S3_USE_PATH_STYLE`
-    # to true configures the S3 client to make requests like
-    # https://hostname.domain/bucket instead.
-    # Defaults to "false"
-    S3_USE_PATH_STYLE: false
-    # Max payload size for each cache object sent by Turborepo
-    # Defaults to 100 MB
-    # Requests larger than that, will get "HTTP 413: Entity Too Large" errors
-    MAX_PAYLOAD_SIZE_IN_MB: "100"
-
-# Now you can run your turborepo tasks and rely on the cache server
-# available in the background to provide previously built artifacts (cache hits)
-# and let Turborepo upload new artifacts when there is a cache miss.
-- name: Run tasks
-  run: turbo run test build typecheck
-```
+You can use the Turbo Cache Server as a **GitHub Action**. Here is how:
+
+1. In your workflow files, add the following global environment variables:
+
+    ```yml
+    env:
+      TURBO_API: "http://127.0.0.1:8585"
+      TURBO_TEAM: "NAME_OF_YOUR_REPO_HERE"
+      # The value of TURBO_TOKEN is irrelevant
+      # as we don't perform any auth against the cache server
+      # but it's still necessary for Turborepo
+      TURBO_TOKEN: "turbo-token"
+    ```
+
+    > [!NOTE]
```

Comment:
gh markdown requires a line break for these to work

Reply:
interesting. There seems to still have issues 🤔
https://github.com/leoweigand/turbo-cache-server/tree/patch-1

![CleanShot 2024-09-10 at 21 04 53@2x](https://github.com/user-attachments/assets/94071ccc-f1d5-4ac3-afab-d43658789f52)

Reply:
ah yeah they can't be inside of a list. moved to the end of this section.